docs/scos/dev/upgradability-services/new-documentation/private-api-class-extended-or-used.md

```diff
@@ -0,0 +1,224 @@
+---
+title: Private API class was extended or used
+description: Reference information for evaluator and upgrader tools.
+last_updated: Mar 23, 2022
+template: concept-topic-template
+---
+
+## Private API class was extended or used
+
+Private API updates can break backward compatibility. So, backward compatibility in minor releases is not guaranteed in the private API. For example, you use a core class or method on the project level. If it is updated or removed, it might change behavior or cause issues.
+
+The following core classes are exceptions, and you can use and extend them on the project level:
```

Comment:
Who provided the list?

Reply:
Ihor, please ignore.

@VladislavStrelchenko please double check :)

Reply:
Hi @helen-laktionova, we also can add: Plugin, Client, Service, to the Particular classes list

Reply:
Fully disagree on the "Plugin"
Lets discuss the whole block